acceptance-tests/src/test/scala/io/renku/graph/acceptancetests/flows/TSProvisioning.scala

```diff
@@ -74,7 +75,9 @@ trait TSProvisioning
       sleep((5 seconds).toMillis)
     }
 
-    `wait for events to be processed`(project.id, accessToken, commitIds.size)
+    val items = eventLogClient.getEvents(project.id.asLeft).unsafeRunSync().size
```

Comment:
Actually, my idea was slightly different. What I imagined was to pass the expected number of events to the `wait for events to be processed` and only there call the new `eventLogClient.getEvents` and check if the expected number equals the size from the `getEvents`.

Reply:
But why would we want to first "guess" and then check? I thought this is more of a helper method to only wait for things so the tests can continue, not really part of the test?